### ISSUE TITLE
[Feature request] plan to position without orientation constraints

### ISSUE THREAD

**Issue by broesdecat** [ros-industrial-attic/industrial_moveit#41]
Hi,

As far as I understand it, industrial_moveit can currently only be used if you have a jointstate goal.
However, this prevents the planner from exploiting any additional freedom in the robot movement, e.g. if only the target position is fixed.
Is there already a way to get this working, or would the code need to be adapted to take also cartesian goals (which is probably significant effort)?

Thx!
Broes

**Comment by jrgnicho** [comment]
@broesdecat indeed, planning for joint goals is very limiting and  planning to cartesian goals is something we've been wanting to incorporate into STOMP.  In my opinion this feature would sample the cartesian goal (in a similar fashion to OMPL) and spawn multiple stomp planning instances until one succeeds.  At the moment no one is working on this feature but if you have any ideas feel free to create a PR and I'd be more than happy to provide guidance.
